ddtrace/bootstrap/sitecustomize.py

```diff
@@ -73,6 +73,7 @@ def drop(module_name):
             "ddtrace",
             "concurrent",
             "typing",
+            "_operator",  # picking issues with typing module
```

Comment:
What's "picking issues"?

Reply:
Ah! It was supposed to read `pickle` 🤦  I don't know how this ended up reading `picking` instead 😊